operator/src/main/helm/coherence/values.yaml

```diff
@@ -170,7 +170,7 @@ store:
   #   be useful for faster cluster start-up in certain scenarios such as testing but could
   #   cause data loss if multiple Pods are stopped in parallel so the default should be
   #   used in production.
-  podManagementPolicy: OrderedReady
+  podManagementPolicy: Parallel
```

Comment:
You need to change the comments too. A few lines above this it still says "The default value is OrderedReady which causes Pods to start and stop in sequence".

Reply:
Thank you, I have updated it.